src/collar/oc_titler.lsl

```diff
@@ -72,20 +73,17 @@ float min_z = 0.25 ; // min height
 float max_z = 1.0 ; // max height
 vector g_vPrimScale = <0.08,0.08,0.4>; // prim size, initial value (z - text offset height)
 
-list ATTACH_POINT_ROTATIONS = [
```

Comment:
This attach point rotation stuff is needed for the new "image" command.

Reply:
agreed, was trying to keep all scripts in all grids up to your latest with changes but missed a few